### PR TITLE
AlphaBravo Pledge

### PR DESCRIPTION
**Pull request by polyackiy**
Agoric Pledge Markdown
moniker: AlphaBravo
email: polyackiy@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
 # Agoric Validator Pledge
```
Comment (jeetraut) [inline README.md:2]:
```suggestion
[AlphaBravo Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper1cvr94kqlp9ss4mx5gkm76h8jf5lnhwvvrav9nd)
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add Explorer Link

**Comment by dckc** [comment]
Hi. Thanks for the contribution, but we need your README.md file to be in a subfolder. Take a look at #2 for an example where the README.md file is `BlockShark/README.md`.

We could try to resolve the conflicts in this branch / PR, but that would be kind of a hassle, so... can we ask you to just make another PR?

**Comment by polyackiy** [comment]
> Hi. Thanks for the contribution, but we need your README.md file to be in a subfolder. Take a look at #2 for an example where the README.md file is `BlockShark/README.md`.
> 
> We could try to resolve the conflicts in this branch / PR, but that would be kind of a hassle, so... can we ask you to just make another PR?

Sorry for taking so long. Another [PR](https://github.com/Agoric/validator-profiles/pull/88)

**Pull request closed without merging**